### ISSUE TITLE
Create an Accessibility Evaluation Test Plan for Releases

### ISSUE THREAD

**Issue by paul-rinaldi** [codeformilwaukee/DECARCERATION-PLATFORM#90]
**Is your feature request related to a problem? Please describe.**
Without testing for accessibility, we cannot be certain our releases our accessible by the population we hope to serve.

**Describe the solution you'd like**
1) A list of tasks (comment below in addition to the list here) to complete to evaluate the accessibility of the site. These tasks should be able to be performed by any user on the site. They should be easily understood that an undergraduate college non-tech savvy student could relate them to a non-tech savvy formerly incarcerated individual.
2) And a list of:
• testing methods to be used, and when they’ll be used in
the process; (probably after a launch)
• how the testing methods will support the production team’s
progress toward the usability targets;
• how the test results will be documented; and
• how the test results will be fed back into the process to
improve the product.

**Describe alternatives you've considered**
There may be some automatic tools to check for alt tags, colors, and or text size not resizing properly.

**Additional context**
Users will likely use phones and desktop computers as was found in our UX customer discovery research.

**Comment by paul-rinaldi** [comment]
Good reference https://2486634c787a971a3554-d983ce57e4c84901daded0f67d5a004f.ssl.cf1.rackcdn.com/campfire-restaurant/media/accessibility-for-everyone-5a5fb4be3ee8b.pdf
